src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -1966,7 +1966,7 @@ export default defineComponent({
         const popUpLayout = seconds > 0
           ? { icon: 'arrow-right', invertContentOrder: true }
           : { icon: 'arrow-left', invertContentOrder: false }
-        const formattedSeconds = Math.abs(seconds)
+        const formattedSeconds = Math.abs(seconds).toFixed(2)
```

Comment:
My solution for avoiding `10.00` from https://github.com/FreeTubeApp/FreeTube/pull/8047
```suggestion
        // `+value` converts string back to float
        const formattedSeconds = +Math.abs(seconds).toFixed(2)
```